### ISSUE TITLE
When all tools of a single type are out of the toolbox, the button should have `aria-disabled: true` attribute

### ISSUE THREAD

**Issue by arouinfar** [phetsims/circuit-construction-kit-common#1206]
When the toolbox looks like this, with all voltmeters and non-contact ammeters in the Play Area:
<img width="155" height="63" alt="Image" src="https://github.com/user-attachments/assets/7fddfb4a-8b5d-444a-b17d-64a5d9e24060" />

The a11y View shows the buttons are stil enabled:
<img width="583" height="253" alt="Image" src="https://github.com/user-attachments/assets/285d18c7-5e94-468c-bb17-0d13942bf1e9" />

We use `aria-disabled: true` in the carousel to communciate when all items of a given type have been removed, such as:

<img width="170" height="88" alt="Image" src="https://github.com/user-attachments/assets/c600e3be-c623-4eee-b050-80ce29cb0e19" />

We should do the same for the Measurement Tools.

@samreid can you please:
- [x] Use `aria-disabled: true` when all measurement tools of a single type have been removed from the toolbox.
- [x] Change the name of the buttons to use the "Add" Prefix, like the carousel items. (Since you'll be in that neighborhood.)
